projects/plugins/jetpack/modules/widget-visibility/editor/index.jsx

```diff
@@ -148,7 +158,7 @@ const VisibilityRule = props => {
 		{ label: __( 'Category', 'jetpack' ), value: 'category' },
 		{ label: __( 'Author', 'jetpack' ), value: 'author' },
 	]
-		.concat( isWpcom ? [] : optionsDisabledOnWpcom )
+		.concat( 'simple' === getSiteType() ? [] : optionsDisabledOnWpcom )
```

Comment:
I could be missing something, but could the existing `isSimpleSite()` util be used here? 

https://github.com/Automattic/jetpack/blob/2e049665899fc74169e7da87131271fa451d1acb/projects/plugins/jetpack/extensions/shared/site-type-utils.js#L27

Reply:
Unfortunately since it's not part of the same bundle (Blocks vs. Widget Visibility), we cannot fully rely on it. That's what I started from here though. In the future, and once we have a shared utility like in #22505, we could use it in both places.

Reply:
Got it, I haven't delved into the webpack setup so just assumed it would be imported and bundled up. Thanks for the explanation 👍

Reply:
Now that this was moved to an external js package, we can import it instead of duplicating it! 🥳 #22741